### PR TITLE
Cryptominded - Funding request - Beginners courses & course platform 

### PR DESCRIPTION
**Pull request by dylandamsma**
# Request for Nest membership and funding

Request for funding to develop a high-quality beginners course & course platform. (Related proposal: [Click here](https://github.com/aragon/nest/issues/4))

**Team name**: Cryptominded

**Proof of concept / research whitepaper**: 

The Cryptominded website gets about 300k unique visitors per month and has been presented in multiple online magazines as 'the go-to place to learn about cryptocurrencies'. Thereby we've spoken to hundreds of beginners whom want to get into the space and outlined a course we think would address most of these questions (link under #proposal)

**Funding request:** $75k _(breakdown below)_

**Legal structure**: 
Private Limited (In Estonia) + Community Foundation in set-up
> (No outside investments, completely bootstrapped and affiliate revenue will be used to further help the cryptocurrency industry through tools & education.)

**The Team:**

1. **Dylan Damsma:** Founder of Cryptominded - Cryptocurrency investor & educator. (NL/BE)
> Co-responsible for written & videographic course content, interviewer & project lead.

2. **Ben Bodien:** Front-end developer & avid cryptocurrency investor (UK)
> Front-end development , course editing & feedback. 

3. **Victor Lim:** Back-end developer & miner (MY)
> Back-end development of platform

## Proposal

At Cryptominded our goal has been to educate beginners on cryptocurrencies from the start. As the platform & industry gained more traction, we've spoken to hundreds of beginning cryptocurrency investors online and offline at events that we've organised and have gotten a broad understanding of beginners' challenges, questions and motives.

We are confident that we're able to bring out a high-quality course to educate the beginning investors & users of cryptocurrency coming into the industry and support a sustainable growth of this market. Over the past months we've been answering many questions from beginners, have helped to fight phishing attacks and educate around security. This definitely has led to us being awfully aware of the need of education in this scene.

While the technology that supports cryptocurrencies is absolutely amazing and evolving with the day, we've seen over and over again that the UX has still a long way to go. Many people fail to understand the basics and end up making crucial mistakes due to this lack of understanding. However, we're of the belief this is mostly a sign that better education is required alongside an improving UX, after all when you give an old person access to internet banking, there's a very good chance that person will make crucial mistakes as well. 

Thereby we believe there will always be the lack of financial education among many, which paired with the quickly rising values of cryptocurrencies results into worrying decisions affecting their personal finance, this is something we would love to help prevent through proper education.

Educating newcomers is really important to us and at the core of why we're spending time on Cryptominded, and while we indeed acknowledge there will always be people that are not willing to put in the time to learn, we have also observed there's a lack of quality education material in this industry which is unbiased and/or freely accessible.

We’re planning to change this by bringing out a high-quality **14 module course on cryptocurrency investing, trading & usage**, addressing questions and concerns for newcomers and helping them to avoid the many pitfalls they have to navigate. In the course we’ll be addressing topics such as:

1. Getting beginners to understand what cryptocurrencies are and why they get so much attention at the moment.
2. Answer the question whether people should invest or not and and help them evaluate investment decisions. 
3. Help people figure out how to purchase, transact & safely store cryptocurrencies,
4. Protect users from common mistakes & scams.
5. Helping beginners understand the differences between various currencies.
6. Answer the most commonly asked questions by beginners.
_A full outline of the course can be provided upon request_

We plan to rope in multiple perspectives and have already lined up multiple interviews with experts in the cryptocurrency, financial history & personal finance industries.

We’re applying for a grant because we would like to put out a high-quality course, one that you’d be comfortable referring your own mom to in case she wants to learn more, without requiring to charge for the course or have biased opinions due to underlying motives.

Producing a high-quality course requires proper copywriting and editing to make the course well designed and the information easily understood. We expect to make videos studio quality, and expect additional costs to travel to meet & interview experts on the topics of cryptocurrencies and personal finance.

We want this course to be about common sense, not 'get rich quick' type of content. It will give a solid educational foundation for people that want to get into cryptocurrencies to either invest in it or use them in some other way - say, using it as a currency.

## Course outline
The course overview can be found here, and we'd be very happy and open to receive feedback on it:
https://docs.google.com/document/d/10ljRqWyTu-PsqEKbLIKSVxuRfHvmUiu3wDQzyG4AenI/edit#

We're more than happy to work with the Aragon community to ensure the quality of the course, such as making sure the content is factually correct, unbiased/balanced, easy to understand for those not as technically inclined as most of the aragon community and ensure we go in-depth and address all perspectives.

## What we will deliver

1. A (forever) **free to consume**, but high-quality, beginners course existing of 14 modules containing multiple lessons both theoretical and practical with the aim to address all the important topics and questions beginners may have.
2. A community forum where questions can be asked, lessons be discussed and where those with more expertise are ready to answer & clarify any questions by beginners.
3. A course on utilising Aragon as entrepreneur to build self-governing communities or run a distributed business.
4. 12 world-wide events to reach & educate beginners and promote the course.
5. A research platform where people can find factual data about all the various cryptocurrencies out there to make better investment decisions and avoid obvious vaporware and/or scams

## Planned Roadmap

1. Jan 2018 - Launch of new community platform
2. End Feb 2018 - BETA Testing beginners course & acquiring feedback
3. March 2018 - Full Launch & promotion of the course V1
4. April 2018 - Adding in exams, tests & interactive elements into the course
5. May 2018 - Localisation of course content
6. July 2018 - Course for starting cryptocurrency entrepreneurs including explanations on Aragon.
7. August 2018 - Launch of the new Cryptominded cryptocurrency research platform.

After this we'll continue to develop additional courses & tools to help people wisely invest in crypto, safely store crypto, use crypto and more of which the costs will hopefully be supported through affiliate income at that point.

## What will funding be used for: 

- Editing, research & translation costs of the course ($7.5k)
- Costs for recording, subtitling & editing videos ($15k)
- Quality checks with experts ($0 - $5000)
- Cover costs for the interviews with experts in the crypto & financial industry to get multiple perspectives ($10k)
- Promotion of the course both online & offline - i.e. through events ($15k)
- Development & server costs for the cryptominded course & research platform ($22.5k)

_These are estimatations of the predicted costs - A transparent spending overview will be provided and remaining resources will be used for additional course content, promotion or development of aiding tools_

## Testimonials

> I believe Cryptominded team is more than up to the task of creating high quality, accessible courses. I was fortunate to work with Dylan and his team, organizing events or video streams - and their work was always of high standards, creative and delivered on time. Dylan's motivation and passion for educating people about personal finance stems from his core values, and is something he truly cares about. Of course it goes without saying that Cryptominded team has all necessary knowledge and skills for the task.

Arseniy Zarechnev, Founder & CEO at Mothership.cx

> Dylan is a highly credible, smart, engaging person deeply interested in helping newcomers learn about cryptocurrency. I'm proud to be his friend and a member of cryptominded.

Terrence Yang, Founder Yang Ventures & cryptocurrency fund advisor

> Dylan and his team at Cryptominded have successfully generated not just a one-stop-shop for cryptocurrency education and resources, but also a vibrant and active community around it. Having spent time with Dylan on his forums and in 1-on-1 video conferences, I have first hand experience of his ability to deconstruct complex subject matter into simple digestible format, and it is this skill which I believe to be key to generating high impact educational material. Dylan and team are uniquely qualified to do this and I wish them all the best.

Oliver von Landsberg-Sadie, CEO & Founder of BitcoinBro.com (Cryptocurrency broker)

-----

Thank you for considering this request for funding. We're looking forward to a partnership with the Aragon Nest fund & the Aragon community

### CODE REVIEW

**Comment by 409H** [comment]
I've met the CryptoMinded team twice now - one time when giving a anti-phishing talk in London (https://www.meetup.com/The-Cryptominded-Meetup-London-Edition/events/242983598/) and another time at an event called CryptoBeers (https://www.meetup.com/The-Cryptominded-Meetup-London-Edition/events/244863174/).

Dylan and team put together some excellent material in not just their meetups, but also their website resources and Slack channels. I can vouch for the CryptoMinded team and brand when I say they can really make an impact in helping onboard and educating people into the cryptocurrency world.

**Comment by dylandamsma** [comment]
@izqui Thanks and I shall discuss that with the team!

**Comment by dylandamsma** [comment]
Hi @izqui & @mariapao - We feel it would be fair to do so, however under some specific conditions. We'd be generally fine open-sourcing the platform code, but are slightly afraid to have someone rebrand the website based on the open-source code and take the content from our site and compete based on that. Which, unfortunately, we have quite a bit of experience with by now...  

We'd be happy to open-source all code under GPL license so that flat out copying the code and distributing an identical copy of the platform ain't possible, but nonetheless everyone can learn from the code and use code from the platform to use in their own applications that serve other purposes as long as it's not a copy-paste replica of the website put out for identical purposes as Cryptominded.

Does that seem fair to you?

**Comment by izqui** [comment]
That sounds reasonable @dylandamsma and was exactly what I was going to reply in our Telegram conversation.

By effective licensing it with a copyleft license you get all the upside of open source while at the same time protecting yourself from others capitalizing on your work. We use GPLv3 for almost all Aragon :)

File: course-fund-request-cryptominded.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Request for Nest membership and funding
```
Comment (izqui) [inline course-fund-request-cryptominded.md]: Actually can you change the directory of the file and also split into team.md and roadmap.md? Reference proposal: https://github.com/aragon/nest/pull/2/files

**Comment by izqui** [comment]
Can you put it all inside a cryptominded directory (`grants/cryptominded`) and remove the `.DS_STORE` file?

**Comment by dylandamsma** [comment]
@izqui should be properly restructured now. Let me know if anything else is required. 

I added comments regarding the investment lesson in the course overview file, which was a concern outed by a few of you. 

We'd like to stress that this course, nor any specific lesson, will not be us telling people to invest in cryptocurrencies in general or any specific tokens for that matter. We want to give people a foundational layer on personal finance and evaluating crypto assets to be able to evaluate their own position, figure out whether they should or should not invest and be able to better separate vaporware/scams from the legitimate projects. Whether a project/token is a good investment or not, is a judgement which should be made by the student him/herself and is not something we will comment on. Nor will we guarantee or make statements that cryptocurrencies will continue to go up and hence investment is a good decision by default. It's the exact kind of content we want to counter by providing an unbiased overview of the pros & cons and by roping in multiple perspectives to provide a foundational layer of personal finance literacy and evaluation skills.

We'd be more than happy to have this content reviewed by Aragon (+ Community) and placeholder before publication, however, I'm convinced addressing this topic is critically important for beginners to understand as this is one of the most commonly asked about topics. 

We want newcomers to understand this is not a market that may only go up and thus investment decisions should be made wisely and that they should be wary of any such statements made by others in the industry.

We've come across courses ourselves that did make such claims to be able to sell more of their course (i.e. Tai Lopez courses on bitcoin investing) and it's deeply worrying that people are consuming such content and following such advice. We want to be the counter voice to content with such statements and I'm convinced we're able to provide this. 

As I personally find it critically important this is being done in an unbiased manner, we're planning interviews with 3rd parties that have varying views, so that it's not just our opinions being pushed onto the student. 

Please let us know if there are any outstanding concerns with regards to this.

**Comment by dylandamsma** [comment]
Before I forget: I'll work on an outline for an additional course in which we'll discuss the Aragon platform (among other tools required), which will be targeted at entrepreneurs, NGOs & community foundations to learn how they can utilise Aragon for their use-case to ensure fair, democratic & transparent business practices.

**Comment by dylandamsma** [comment]
Through discussion with Aragon, we've come to the decision to work on an additional course (freely consumable) that will teach people about, what we termed, Cryptopreneurship. Entrepreneurship in the blockchain / cryptocurrency industry and the entire ecosystem that comes with it. 

Our reasoning here is that people who're getting into cryptocurrencies for mainly financial reasons tend to be fairly entrepreneurial. This means our audience is perfectly suitable for such a course, which also presents us the opportunity to teach people about Aragon in-depth and help business understand the power of using it, among other technologies and innovations, to empower their business models.

We believe helping people understand the ecosystem from an entrepreneurial point of view and show them the importance of tokens, decentralisations and extra security provided by blockchain technology will help them realise the huge potential these services & technologies offer for them.

Aragon plays a big part in this ecosystem for entrepreneurs and therefore would be central in the course. The below outline is a rough outline, which we'd love to further explore with entrepreneurs in the space.

**Course:** Become a Cryptopreneur
- The importance of decentralization and efficiency for your organization (general overview)
- Fundraising with transparency (aragon)
- Managing stakeholders with voting through tokens with Aragon
- Identity management with uPort
- How to grow your community and get them involved
- Invoicing, paying and bookkeeping with cryptocurrencies (Request / Aragon)
- Financial reporting made easy (Aragon)
- Building your unstoppable organization

**Planned:** May / July 2018

While we do understand the beginner's course we have proposed does not have a lot of overlap with what Aragon offers as a solution, we do still find it critically important such course is being created. We believe that we'd be able to make these two courses upon receiving the grant and be able to fund extra costs we occur ourselves if deemed necessary to maintain quality.

**Comment by cburniske** [comment]
Thanks for the proposals and all the work, Dylan. As we've discussed your materials, we've realized we'd like for all educational material funded by Aragon to be focused on increasing the utility, or adopting the utility, of Aragon and its surrounding protocols. We would like to leave it to businesses (or protocols) that depend on investing / speculating to fund any education around those topics. 

To that end, we'd like to scrap the proposed syllabus on "14 module course on cryptocurrency investing, trading & usage" (sorry!), but would like a similarly detailed syllabus to the end of what you've discussed above.  

Also, another idea for such a course could be: "10 Organizations You Can Build on Aragon." If you were to do 10 videos, each one an hour or so that are kind of like this: https://www.youtube.com/watch?v=RKzMT8Z24X4, then I think that would be really cool and would be high value-add to the library of Aragon content and appeal to a diverse array of folks that could use Aragon to run their organizations -- things could be as random as a gaming organization, for fantasy football or leagues of other kinds, but should be accessible to the everyday user. 

If we don't want it to be exclusively focused on Aragon, then what you've created will work just fine, but i do have a question regarding the intent of your outline for Cryptopreneur: do you intend for the most value to be derived if people watch all 10 videos, or will these 10 video be digested here and there, but not necessarily building one upon the other for a specific person? these aren't leading questions, i'm just openly raising them

As a last note, we are likely to give a $7.5K for the first video, as a trial run, and then go from there with the remainder of the grant. Happy to talk about any of this offline, too

**Comment by dylandamsma** [comment]
Hi Chris, I'll get back to you as soon as possible!

**Comment by cburniske** [comment]
Dylan, I discussed with Tatu & John from the team and before you get too far into designing the syllabus it might be good to chat and decide on the first video, so we can do a test-run together. AragonOS 0.5 is going to be released soon, and we are going to have some specific needs for content, which could tie into a series - the Aragon team communicates as a group most frequently through Keybase, if you could find me there @cburniske

**Comment by dylandamsma** [comment]
Hi @cburniske, I contacted you via Keybase.

**Comment by cburniske** [comment]
thanks, @dylandamsma -- we're going to close this grant as we have a limited number of grants to give for the "original class" and would like to focus them on developer teams building necessary infrastructure. we really appreciate the thought and work you put into things, and do have some ideas for you around content that Aragon needs, and thus are happy to work on more of a contractor basis for which we'll discuss in keybase

**Comment by DaniellMesquita** [comment]
@dylandamsma
This course can be added to an education platform: https://github.com/aragon/nest/issues/123

**Pull request closed without merging**